docs/DEPLOYMENT.md

```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
 
 ## Tag Release ##
 
-On the Monday after the milestone closes, Tag the release.
+This will happen on Friday at the end of sprint.
 
 1. https://github.com/mozilla/testpilot/releases/new
 2. Tag Version: YYYY-MM-DD (append -N if more than one release is tagged on a given day: 2016-04-08-1)
 3. Release Title: YYYY-MM-DD
 4. Click `Publish`
 5. Wait for CircleCI to confirm that the release has passed ([tags uploaded to container](https://hub.docker.com/r/mozilla/testpilot/tags/)).
 
+Please be as detailed as possible in the release notes.
+
 ## Request Deployment ##
 
+This will happen on Friday at the end of sprint.
+
 Once we've confirmed our release passed CircleCI, and we've confirmed that it is uploaded into the container, we need to request for Ops to deploy to stage.
 
 This is done via a deployment bug.
 
 Example: https://bugzilla.mozilla.org/show_bug.cgi?id=1263682
 
+Include Softvision in the CC:
+
 ## Deployed Stage ##
 
+This will happen on Friday at the end of sprint.
+
 Ops will inform us once we are deployed to our *staging environment*: [http://testpilot.stage.mozaws.net](http://testpilot.stage.mozaws.net).
 
 ## Test Stage ##
 
+This will happen on Friday at the end of sprint.
+
 Send out an email notification to `testpilot-dev@mozilla.com` to please test the staging environment.
```

Comment:
Is there a good example email that could be included? Archives are here: https://mail.mozilla.org/pipermail/testpilot-dev/